rust/common-runtime/scripts/generate-artifacts.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Great solution for the recursive cargo builds with our wasm artifacts :partying_face: